#goal
geographic_msgs/GeoPose[] poses
float64 acceptance_radius
float64 acceptance_yaw_thresh
int64 stable_counts_ms
---
#result
bool success
---
#feedback
int64 stable_counts
float64 lin_error
float64 ang_error
